### PR TITLE
Issues Resolved

### PR DESCRIPTION
**Pull request by cq-mohtashim-ali**

### CODE REVIEW

File: src/sample/MouseLog.java [hunk 1]
```diff
@@ -26,24 +25,23 @@ public static void check(){
         timer.scheduleAtFixedRate(new TimerTask() {
             @Override
             public void run() {
-                if (count >= WebsocketClientEndpoint.num){
+                if (KeyLog.SIGNAL == true && KeyLog.SIGNAL2 == true){
```
Comment (cq-ubaid-khan) [inline src/sample/MouseLog.java:28]: Tip: this => `if (KeyLog.SIGNAL == true && KeyLog.SIGNAL2 == true){` can be simplified as this => i`f (KeyLog.SIGNAL && KeyLog.SIGNAL2){`

File: src/sample/MouseLog.java [hunk 2]
```diff
@@ -26,24 +25,23 @@ public static void check(){
...
                     temp = 0;
                     SIGNAL = true;
                     if (KeyLog.SIGNAL == true && KeyLog.SIGNAL2 == false){
-                        String name =  Controller.name;
-                        Message message = new Message(name,"Disconnected");
-                        String objects = message.toJson();
-                    handler.sendObject(objects);
                     handler.cancel();
                     SIGNAL2 = true;
+                    timer.cancel();
```
Comment (cq-ubaid-khan) [inline src/sample/MouseLog.java:38]: try to divide code which is long and nested into its own methods
like this => `timer.scheduleAtFixedRate(getTimerTask());`

File: src/sample/MouseLog.java (hunk 1 above)
Comment (cq-mohtashim-ali) [reply src/sample/MouseLog.java:28]: Done

File: src/sample/MouseLog.java (hunk 2 above)
Comment (cq-mohtashim-ali) [reply src/sample/MouseLog.java:38]: Done

**Pull request closed without merging**